upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua

```diff
@@ -4033,7 +4036,7 @@ else
 				local x, y = 0, 0;
 				
 				surface.SetMaterial(Clockwork.ScreenBlur);
-				surface.SetDrawColor(255, 255, 255, 255);
```

Comment:
This should be SetDrawColor(color_white), did you do a Find and Replace or something? 😨 There are quite a few instances of this mistake in this file.

Reply:
To clarify, the mistake is that it was changed to `SetDrawcolor_white`.

Reply:
oopsie woopsie

Reply:
[Well actually -unless i'm missing something here- SetDrawColor(kshort, kshort, kshort) is faster than SetDrawColor(color)](https://i.imgur.com/kyR643Q.png)

I didn't compare the two different type of calls before, maybe I should just have stripped Color().

Reply:
whatever i'm lazy

Reply:
That’s a shame 😥